dagfactory/dagbuilder.py

```diff
@@ -771,7 +774,11 @@ def configure_schedule(dag_params: Dict[str, Any], dag_kwargs: Dict[str, Any]) -
                 if has_datasets_attr:
                     schedule.pop("datasets")
         else:
-            dag_kwargs["schedule"] = dag_params.get("schedule")
+            schedule = dag_params.get("schedule")
+            if schedule == "None":
```

Comment:
```suggestion
            if schedule.strip().lower() == "none":
```

Reply:
Fixed in 4ce9d03fea05a8d876de06c2fd1d2f5dcaaa25cf